Sources/Clients/GatewayAPI/GatewayAPIClient/GatewayAPIClient+Live.swift

```diff
@@ -140,7 +140,6 @@ extension GatewayAPIClient {
 				request: GatewayAPI.StateEntityDetailsRequest(
 					atLedgerState: ledgerState?.selector,
 					optIns: .init(
-						nonFungibleIncludeNfids: true,
```

Comment:
Ids are no longer fetched on the initial request, those will be fetched on demand when viewing the nft collection.